examples/pipelineruns/conditional-pipelinerun.yaml

```diff
@@ -37,8 +36,7 @@ spec:
   steps:
     - name: run-ls
       image: ubuntu
-      command: ["/bin/bash"]
-      args: ['-c', 'ls -al $(inputs.resources.workspace.path)']
+      script: 'ls -al $(inputs.resources.workspace.path)'
```

Comment:
Could we make here as well: https://github.com/tektoncd/pipeline/blob/master/examples/pipelineruns/conditional-pipelinerun-with-optional-resources.yaml#L2